code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -4196,7 +4196,7 @@ datum
 					O.setMaterial(getMaterial("pizza"))
 
 		fooddrink/friedessence
-			name = "The Physical Manifestation Of The Very Concept Of Fried Food"
+			name = "the physical manifestation of the very concept of fried food"
```

Comment:
definitely needs to be capitalized

Reply:
I think there's some value to be had from the deadpan delivery the lowercase title implies, personally

Reply:
I think there's some value to be had from the deadpan delivery the lowercase title implies, personally